src/CoreWCF.Primitives/src/CoreWCF.Primitives.targets

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <ItemGroup Condition="'$(_CoreWCFSuppressWebApplicationExtensions)' != 'true' AND
```

Comment:
Can `$(_CoreWCFSuppressWebApplicationExtensions)` be `$(CoreWCFSuppressWebApplicationExtensions)` (without `_`) ?

Reply:
Generally properties starting with an underscore are for internal purposes and aren't intended as a build api. My thinking here is that it's not intended as something people need to be using and is what I call a "get out of jail free" card that we only inform people of if there's any further problems we didn't expect, with the intention it would be a workaround until we fixed whatever problem they are having.  
I don't feel strongly about this, so if you still feel it shouldn't have the underscore, then I'll remove it.

Reply:
Ok i was wondering if the _ prefix is a magic feature of msbuild but I see this is just a naming convention. That's fine to me to keep it private as it should not be used in theory.